### PR TITLE
Enable linting on GitHub Actions.

### PR DESCRIPTION
**Pull request by jianjunz**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -3,6 +3,16 @@ name: Continuous Integration
 on: [push, pull_request]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```
Comment (taste1981) [inline .github/workflows/ci.yaml:9]: add a name before each step.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -3,6 +3,16 @@ name: Continuous Integration
...
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+    - run: npm install
```
Comment (taste1981) [inline .github/workflows/ci.yaml:13]: Add a name before each run.

**Comment by jianjunz** [comment]
I don't think it's a common practice to add a name for every step, especially for simple and well known commands, e.g.: https://github.com/w3c/mediacapture-main/blob/main/.github/workflows/auto-publish.yml

**Comment by taste1981** [comment]
> I don't think it's a common practice to add a name for every step, especially for simple and well known commands, e.g.: https://github.com/w3c/mediacapture-main/blob/main/.github/workflows/auto-publish.yml

Github's own document as a reference: https://github.com/marketplace/actions/lint-action

**Comment by jianjunz** [comment]
Name is an optional field. It will be displayed on GitHub actions report. Since `checkout`, `setup-node` and `npm install` are very popular for node.js applications, I think "Run actions/checkout@v2" is easy enough for readers to understand.

**Review by taste1981 — APPROVED** [review]

**Pull request merged**